package.json

```diff
@@ -12,6 +12,10 @@
   "bin": {
     "run": "bin/run.js"
   },
+  "scripts": {
+    "build": "babel src/ --out-dir lib/",
+    "test": "mocha"
```

Comment:
Instead of adding `script` line to travis we could add `"pretest": "npm run build"` to scripts